PublishParasoftTestResults/publishtestresults.ts

```diff
@@ -269,9 +279,44 @@ function checkStaticAnalysisViolations(sarifReports: string[], index: number) {
     }
 }
 
+function loadSettings(localSettingsPath : string) {
+    if (isNullOrWhitespace(localSettingsPath)) {
+        tl.warning('Local settings file path is not specified.');
+        return null;
+    }
+
+    let localSettingsFile = tl.resolve(tl.cwd(), localSettingsPath);
+    tl.debug('Path to local settings is ' + localSettingsFile);
+
+    const props = loadProperties(localSettingsFile);
+    if (props === null) {
```

Comment:
Is it possible for `props` to be null but not go into any catch block in `loadProperties`? 
If so, let's move the warning message inside the try block. 
If not, let's remove it since it may be redundant with the error messages printed at the same time.

Reply:
Actually, I think it's better to print out this warning message when there's no error but properties file is empty.